guides/common/modules/proc_synchronizing-smart-proxy-directly-from-red-hat-cdn-by-using-web-ui.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="synchronizing-{smart-proxy-context}-directory-from-red-hat-cdn-by-using-web-ui"]
+= Synchronizing {SmartProxy} directly from Red{nbsp}Hat CDN by using {ProjectWebUI}
+
+[role="_abstract"]
+You can use simplified alternate content sources to configure your {SmartProxy} to sync content directly from Red{nbsp}Hat CDN.
```

Comment:
```suggestion
You can use simplified alternate content sources to configure your {SmartProxy} to sync content directly from Red{nbsp}Hat CDN instead of {ProjectServer}.
```